feg/gateway/services/s8_proxy/servicers/senders.go

```diff
@@ -38,6 +38,8 @@ func (s *S8Proxy) sendAndReceiveCreateSession(
 		cPgwUDPAddr.String(), csReq.String())
 	glog.V(2).Infof("Send Create Session Request (gtp) to %s:\n%s",
 		cPgwUDPAddr.String(), csReqMsg.(*message.CreateSessionRequest).String())
+	//glog.V(4).Infof("Send Create Session Request (gtp) to %s:\n%s",
```

Comment:
Remove?

Reply:
yep i am waiting for one pr I submitted at go-gtp to enable that. 

Will remove like 39 probably and keep 41 (uncommented)

Reply:
ok for now it will stay and will remove in the next PR.
It looks like the new tag is not there but i want to have it handy in place for testing for now